test/WebSites/LocalizationWebSite/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,3 @@
+﻿index
```

Comment:
Could you additionally add a layout (with partial path specification: `Layout = "MyLayout";`) and verify it uses the expander?